### PR TITLE
Validate loaded data

### PR DESCRIPTION
**Pull request by colin99d**
Small PR to validate loaded data.

### CODE REVIEW

File: gamestonk_terminal/portfolio/portfolio_model.py [hunk 1]
```diff
@@ -74,6 +74,28 @@ def load_df(name: str) -> pd.DataFrame:
...
+        for item in ["Quantity", "Price", "Fees", "Premium"]:
+            result = any(df[item] < 0)
+            if result:
+                print(
+                    f"The column '{item}' has a negative value. Ensure all values are positive."
+                )
+                return pd.DataFrame()
+
+        if len(df[~df["Type"].isin(["cash", "stock"])].index):
+            print("Warning: 'Type' other than 'cash' and 'stock' will be ignored.")
```
Comment (DidierRLopes) [inline gamestonk_terminal/portfolio/portfolio_model.py:87]: I think you are missing a `return pd.DataFrame()` here

File: gamestonk_terminal/portfolio/portfolio_model.py [hunk 2]
```diff
@@ -74,6 +74,28 @@ def load_df(name: str) -> pd.DataFrame:
...
+
+        for item in ["Quantity", "Price", "Fees", "Premium"]:
+            result = any(df[item] < 0)
+            if result:
+                print(
+                    f"The column '{item}' has a negative value. Ensure all values are positive."
+                )
+                return pd.DataFrame()
+
+        if len(df[~df["Type"].isin(["cash", "stock"])].index):
```
Comment (DidierRLopes) [inline gamestonk_terminal/portfolio/portfolio_model.py:86]: Another way I would have done this if it was me would have been:
```
if sum(df["Type"].isin(["cash", "stock"]).values) != len(df)
```

When you do `df["Type"].isin(["cash", "stock"]).values` these values become `True` or `False`, when you sum them `False = 0` and `True = 1`. Therefore, if the sum of these is equal to the length of the data frame it means all values were `cash or stock`.

But what you have works :)

File: gamestonk_terminal/portfolio/portfolio_model.py (hunk 1 above)
Comment (colin99d) [reply gamestonk_terminal/portfolio/portfolio_model.py:87]: I still want this to work as normal, I just want the user to know other assets will not be analyzed.

Comment (DidierRLopes) [reply gamestonk_terminal/portfolio/portfolio_model.py:87]: Gotcha!

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**